tests/Feature/AdminTest.php

```diff
@@ -1,442 +1,370 @@
-            'submitted_at' => now(),
-            'approved_at' => now(),
-            'is_pinned' => true,
-        ]);
+    $this->put("/admin/articles/{$article->slug()}/pinned");
 
-        $this->loginAsModerator();
+    expect($article->fresh()->isPinned())->toBeFalse();
+});
 
-        $this->put("/admin/articles/{$article->slug()}/pinned");
+test('moderators can unpin articles', function () {
+    $article = Article::factory()->create([
+        'submitted_at' => now(),
+        'approved_at' => now(),
+        'is_pinned' => true,
+    ]);
 
-        $this->assertFalse($article->fresh()->isPinned());
-    }
+    $this->loginAsModerator();
 
-    /** @test */
-    public function users_cannot_unpin_articles()
-    {
-        $article = Article::factory()->create([
-            'submitted_at' => now(),
-            'approved_at' => now(),
-            'is_pinned' => true,
-        ]);
+    $this->put("/admin/articles/{$article->slug()}/pinned");
 
-        $this->login();
+    expect($article->fresh()->isPinned())->toBeFalse();
+});
 
-        $this->put("/admin/articles/{$article->slug()}/pinned");
+test('users cannot unpin articles', function () {
+    $article = Article::factory()->create([
+        'submitted_at' => now(),
+        'approved_at' => now(),
+        'is_pinned' => true,
+    ]);
 
-        $this->assertTrue($article->fresh()->isPinned());
-    }
+    $this->login();
 
-    /** @test */
-    public function guests_cannot_unpin_articles()
-    {
-        $article = Article::factory()->create([
-            'submitted_at' => now(),
-            'approved_at' => now(),
-            'is_pinned' => true,
-        ]);
+    $this->put("/admin/articles/{$article->slug()}/pinned");
 
-        $this->put("/admin/articles/{$article->slug()}/pinned");
+    expect($article->fresh()->isPinned())->toBeTrue();
+});
+
+test('guests cannot unpin articles', function () {
+    $article = Article::factory()->create([
+        'submitted_at' => now(),
+        'approved_at' => now(),
+        'is_pinned' => true,
+    ]);
+
+    $this->put("/admin/articles/{$article->slug()}/pinned");
+
+    expect($article->fresh()->isPinned())->toBeTrue();
+});
+
+// Helpers
+function assertCanSeeTheUserOverview()
+{
+    User::factory()->create(['name' => 'Freek Murze']);
+    User::factory()->create(['name' => 'Frederick Vanbrabant']);
+
+    $this->visit('/admin')
+        ->see('Freek Murze')
+        ->see('Frederick Vanbrabant');
+}
+
+function assertCanBanUsers()
+{
+    $user = User::factory()->create(['name' => 'Freek Murze']);
+
+    $this->put('/admin/users/'.$user->username().'/ban')
+        ->assertRedirectedTo('/user/'.$user->username());
+
+    $this->notSeeInDatabase('users', ['id' => $user->id(), 'banned_at' => null]);
+}
+
+function assertCanUnbanUsers()
+{
+    $user = User::factory()->create(['name' => 'Freek Murze', 'banned_at' => Carbon::now()]);
+
+    $this->put('/admin/users/'.$user->username().'/unban')
```

Comment:
On helpers, the `$this->put` needs to be replaced by `$this->put`.

Reply:
Got it. I'll definitely update this in the Pest Converter.